neo/app/actor/RequestRouter.scala

```diff
@@ -0,0 +1,72 @@
+package actor
+
+import akka.actor.{Actor, ActorLogging, ActorRef, Props}
+import akka.stream.Materializer
+import edu.uci.ics.cloudberry.zion.actor.BerryClient
+import edu.uci.ics.cloudberry.zion.actor.BerryClient._
+import edu.uci.ics.cloudberry.zion.common.Config
+import play.api.libs.json._
+import play.Logger
+
+import scala.concurrent.ExecutionContext
+
+class RequestRouter (out: ActorRef, berryClientProp: Props, config: Config)
+                    (implicit ec: ExecutionContext, implicit val materializer: Materializer) extends Actor with ActorLogging {
+
+  import RequestRouter._
+
+  val streamingBerryClient = context.actorOf(berryClientProp)
+  val nonStreamingBerryClient = context.actorOf(berryClientProp)
```

Comment:
I see. 
in that case, you can use `forward` function to be clear 😄 
But, as explained earlier, we'd better to clearly deliver the msg by directly give out to berry.